### DISCUSSION TITLE
Usage with next/link

### DISCUSSION THREAD

**Discussion by mxmtsk** [vinissimus/next-translate#665] [Q&A]
I'm trying to translate a string, that has a page to an internal page so I tried this:

```
<Trans
  i18nKey="election:privacyNote"
  components={[
    <Link
      href="/de/page/datenschutz"
      key="link"
      passHref
    />,
  // eslint-disable-next-line jsx-a11y/anchor-has-content
    <a
      className="font-normal text-underline"
      key="hyperlink"
    />,
  ]}
/>
```

but this throws the following error:

> Error: Multiple children were passed to <Link> with `href` of `/de/page/datenschutz` but only one child is supported https://nextjs.org/docs/messages/link-multiple-children

The string to translate is this:

> Mit einem Klick auf Starten« akzeptierst du unsere <0><1>Datenschutzbedingungen</1></0>.

Is this not the correct way of doing it?

**Comment by VictorAssis** [comment]
You can create a component to encapsulate next/link:

```
export const CustomLink = ({ href, children, ...props }) => {
  return (
    <Link href={href || ''} >
      <a {...props}>{children}</a>
    </Link>
  );
};
```

Then you can use with Trans (and on other places):
```
<Trans
  i18nKey="election:privacyNote"
  components={[
    <CustomLink
      href="/de/page/datenschutz"
      className="font-normal text-underline"
    />,
  ]}
/>
```

Remember to change the string:

> Mit einem Klick auf Starten« akzeptierst du unsere <0>Datenschutzbedingungen</0>.

**Comment by aralroca** [comment]
> ```
> <Trans
>   i18nKey="election:privacyNote"
>   components={[
>     <Link
>       href="/de/page/datenschutz"
>       key="link"
>       passHref
>     />,
>   // eslint-disable-next-line jsx-a11y/anchor-has-content
>     <a
>       className="font-normal text-underline"
>       key="hyperlink"
>     />,
>   ]}
> />
> ```

@mxmtsk The `a` should be inside `Link`.

Instead of:

```jsx
<Link
  href="/de/page/datenschutz"
  key="link"
  passHref
/>
<a
  className="font-normal text-underline"
  key="hyperlink"
 />
```

Try

```jsx
<Link
  href="/de/page/datenschutz"
  key="link"
  passHref >
  <a
    className="font-normal text-underline"
    key="hyperlink"
   />
</Link>
```
